src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1812,8 +1814,8 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserInSectionBy
 
         FeedbackSessionResultsBundle results =
                 new FeedbackSessionResultsBundle(
-                        session, responses, relevantQuestions,
```

Comment:
Even though this method is only called from the student pages now, I think you should initialise the value of <code>sectionTeamNameTable</code> in this branch, because this method looks like it should be a generic method that the instructor pages should be able to use too.